lib/internal/errors.js

```diff
@@ -1370,8 +1370,7 @@ E('ERR_INVALID_SYNC_FORK_INPUT',
 E('ERR_INVALID_THIS', 'Value of "this" must be of type %s', TypeError);
 E('ERR_INVALID_TUPLE', '%s must be an iterable %s tuple', TypeError);
 E('ERR_INVALID_URI', 'URI malformed', URIError);
-E('ERR_INVALID_URL', function(input) {
-  this.input = input;
+E('ERR_INVALID_URL', function() {
```

Comment:
if you dont assign anymore input to this.input then the function is not necessary anymore. Jjust replace the function with 'Invalid URL'.

Reply:
Makes sense! Thanks.